pegasus/helpers/section_api_helpers.rb

```diff
@@ -33,6 +33,7 @@ def self.create(params)
         user_type: 'student',
         provider: 'sponsored',
         gender: gender,
+        properties: {sharing_disabled: sharing_disabled}.to_json,
```

Comment:
Why bury `sharing_disabled` in `properties?

Reply:
It currently lives in properties on the `User` model. 

This was a decision we made when we originally implemented this feature. I think we were to start from scratch with `User` we would put many more things on properties and not on the model directly. Or at least in a separate `Properties` model. 
History in some original PRs (tech spec linked): 
https://github.com/code-dot-org/code-dot-org/pull/17064
https://github.com/code-dot-org/code-dot-org/pull/17162

Reply:
Aha. That makes sense. Thanks for the context.